workflow/landing-page/style.css

```diff
@@ -0,0 +1,551 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  transition: all 0.5s ease 0s;
+}
+
+body {
+  background-color: #efefef;
+}
+
+a {
+  text-decoration: none
+}
+
+a:hover {
+  color: black;
+}
+
+h2 {
+  font-size: 1.5em;
+  font-weight: 400;
+  padding: 70px 0;
+}
+
+h3 {
+  margin-bottom: 25px;
+  font-weight: normal;
+  font-size: 1.125em;
+}
+
+.primary-text {
+  font-size: 0.875em;
+  color: #778899;
+  line-height: 1.5em;
+}
+
+li {
+  color: white;
+  font-size: 17.5px;
+  list-style-type: none;
+  font-family: 'Roboto';
+  font-weight: 400;
+  padding: 15px 0px;
+  margin: 0 25px;
+}
+
+li:first-child {
+  padding-top: 25px;
+}
+
+li:not(:first-child) {
+  border-top: 1px solid #474747;
+}
+
+label {
+  width: 130px;
+}
+
+.button {
+  width: 36px;
+  height: 36px;
+  border: 1px solid #d0d0d0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: 0.8em;
+  cursor: pointer;
+  margin: 2px;
+  transition: all 0.5s ease 0s;
+}
+
+.roboto-font {
+  font-family: 'Roboto';
+  font-weight: 400;
+}
+
+.orange-color {
+  color: orange
+}
+
+.blue-color {
+  color: #00bfff;
+}
+
+.green-color {
+  color: #adff2f;
+}
+
+.white-background-color {
+  background-color: #fff;
+}
+
+.content-width {
+  padding: 0 15px;
+  width: 100%;
+}
+
+@media (min-width: 769px) {
+  .content-width {
+    width: 720px;
+    padding: 0;
+    margin: auto;
+  }
+}
+
+@media (min-width: 993px) {
+  .content-width {
+    width: 940px;
+    padding: 0;
+  }
+}
+
+@media (min-width: 1201px) {
+  .content-width {
+    width: 1140px;
+    padding: 0;
+    margin: auto;
+  }
+}
+
+.title {
+  font-size: 2em;
+  font-weight: 700;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 30px 0 20px;
+  font-size: 0.9em;
+}
+
+@media (min-width: 769px) {
+  .header {
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 40px 20px;
+  }
+}
+
+@media (min-width: 993px) {
+  .header {
+    padding: 40px 60px;
+  }
+}
+
+.social-buttons-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+@media (min-width: 769px) {
+  .social-buttons-wrapper {
+    margin-top: 0;
+  }
+}
+
+.button-wrapper:hover {
+  border: 1px solid black;
+}
```

Comment:
I suppose it have to be just for `.button` class, as there's no `.button-wrapper`